install/install.sh

```diff
@@ -82,6 +82,19 @@ fetch() {
   fi
 }
 
+install() {
+ local sudo 
+ if [ -w $BIN_DIR ]; then
+	 sudo=''
+ else
+	 info "Root privileges are required to proceed the installation"
+	 sudo='sudo'
+ fi
+ fetch "${URL}" \
+	| ${sudo} tar xzf${VERBOSE} - \
```

Comment:
I remember some users having trouble with piping stuff directly into sudo and the script being treated as password - maybe do `sudo -v` first here? Correct me if I'm wrong about the semantics here.
Might also want to handle dismissed sudo in a loop or with `exit 1`.
Not sure if this is a priority but maybe missing `sudo` should be handled too either with early exit or fallback to `su`.

Reply:
> I remember some users having trouble with piping stuff directly into sudo

Recenttly?

>  maybe do `sudo -v` first here?

Yes! That will be the update

> dismissed sudo

Do you mean command not found?

> fallback to `su`

I'm not sure if I like it cause *su* requires *root* password,  as fallback I preffer to give a message to user provide the bin location for this specific case

Reply:
> Recenttly?

I think it was the issue in #810.

> Do you mean command not found?

Command not found is an issue, but with "dismissed" I meant stuff like failing to provide the proper password, `ctrl+c` and not being in the sudoers file.

> I'm not sure if I like it cause su requires root password, as fallback we can give a message to guide user the needed to provide a bin location for this specific case
What are your thoughts?

I think a message is sufficient since every out-of-the-box setup should have sudo available.